### ISSUE TITLE
plaintext response is not displayed, response throbber does not stop

### ISSUE THREAD

**Issue by demogorgon** [swagger-api/swagger-editor#396]
When I press the "try it now" button for my post request and if the server responds with text/plain message, rather than json, the section with response content does not show up and the throbber keeps on going.

I expect the response container to show the response text and headers, as it normally does

**Comment by mohsen1** [comment]
Don't you see anything in any of the tabs? (Rendered, Pretty and Raw)

**Comment by mohsen1** [comment]
If angular sanitizer fail to parse the response this will happen. With this Swagger and [https://www.ietf.org/rfc/rfc4627.txt](https://www.ietf.org/rfc/rfc4627.txt) url it's possible to duplicate this issue 

```yaml
---
swagger: '2.0'
info:
  version: 0.0.0
  title: Simple API
schemes:
  - https
host: cors-it.herokuapp.com
paths:
  /:
    get:
      parameters:
        - name: url
          in: query
          description: The URL to fetch
          type: string
        
      responses:
        200:
          description: OK
# https://cors-it.herokuapp.com/?url=https://www.ietf.org/rfc/rfc4627.txt
```

**Comment by mohsen1** [comment]
Fixed by #430
